lib/instances.js

```diff
@@ -157,10 +158,15 @@ exports.getControllerKey = async ({id}) => {
 
 // set issuer and capabilities (zcaps) for an instance
 exports.setIssuer = async ({id, controller, issuer, zcaps}) => {
```

Comment:
This should be addressed in a different PR.

```suggestion
exports.setIssuerInstance = async ({id, controller, issuer, zcaps}) => {
```

Reply:
We're in the `instances` API here, isn't the `Instance` you tacked on the end implicit?

Reply:
I agree with @mattcollier

Reply:
I did not realize the file was named appropriately. This suggestion should be ignored.